modules/drv-parts/writers/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  writers = config.deps.callPackage ../../../pkgs/writers {};
+in {
+  imports = [
+    ./interface.nix
+  ];
```

Comment:
I think it would be nice if this would import `dream2nix.modules.drv-parts.core` and thereby become usable as a drv-parts submodule like all our other drv-parts.

So basically instead of defining `config.writers`, the module could define options for the actual interface of the writer and `config.public` for the result.

Reply:
A right, that's a good idea. Just did this quickly while working on a third module including writers. 

Later remembered that I still got an un-rebased https://github.com/nix-community/dream2nix/pull/548 lying around. Will merge those to and use the core interface.